hw_16_four_in_one/js/4in1.js

```diff
@@ -0,0 +1,51 @@
+// Дано масив з елементами різних типів. Створити функцію яка вираховує середнє арифметичне лише числових елементів даного масиву.
+
+let arr =  [1,'asd',2,{},3,4,20,'', 15];
+let barr = arr;
+for (let i = arr.length - 1; i >= 0; i--) {
+  if (typeof arr[i] !== "number")
+    arr.splice(i, 1);
+}
+console.log(barr);
+function numAverage(barr) {
+    let i = 0;
+    for (const arrayElement of barr) {
+        i += arrayElement;
+    }
+    return i / barr.length;
+} 
+    const average = numAverage(barr)
+    console.log(average);
+
+// Написати функцію doMath(x, znak, y), яка отримує 3 аргументи: числа x і y, рядок znak. У змінній znak може бути: +, -, *, /, %, ^ (ступінь ).Вивести результат математичної дії, вказаної в змінній znak.Обидва числа і знак виходять від користувача.
+
+function doMath(x, znak, y) {
+  if (~znak.search(/[^+\-*/%^]/) || znak.length !== 1) return 'Error';
+  return eval(x + znak.replace('^', '**') + y)
```

Comment:
Don't use `eval` method
The task was to realize functions that solve operations. Like it was on the lesson